src/main/java/com/booksaw/betterTeams/util/WebhookHandler.java

```diff
@@ -0,0 +1,84 @@
+package com.booksaw.betterTeams.util;
+
+import com.booksaw.betterTeams.Main;
+import com.booksaw.betterTeams.Team;
+import com.booksaw.betterTeams.TeamPlayer;
+import com.booksaw.betterTeams.customEvents.*;
+import dev.ceymikey.injection.DiscordPayload;
+import dev.ceymikey.injection.EmbedBuilder;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+/**
+ * This is where we listen for the events and define the info for the webhook.
+ * There is some webhook events available not for all event
+ *
+ * @author ceymikey
+ */
+public class WebhookHandler implements Listener {
+
+    private final String configURL = Main.plugin.getConfig().getString("hookURL");
+    private final Boolean createHook = Main.plugin.getConfig().getBoolean("create-hook");
+    private final Boolean disbandHook = Main.plugin.getConfig().getBoolean("disband-hook");
+    private final Boolean pLeaveHook = Main.plugin.getConfig().getBoolean("player-left-hook");
+    private final Boolean teamNameHook = Main.plugin.getConfig().getBoolean("team-nameChange-hook");
+    private final Boolean teamChatEvent = Main.plugin.getConfig().getBoolean("team-chat");
+    private Team team;
+    private TeamPlayer teamPlayer;
+    private String newTeamName;
+
+    @EventHandler
+    public void onTeamCreate(CreateTeamEvent e) {
+        if (createHook) {
+            team = e.getTeam();
+            sendWebhookMessage("TEAM CREATED", "Somebody created the team " + team.getName());
```

Comment:
as an FYI, someone made a PR so you can actually get team names from the TeamCreateEvent and TeamDisbandEvent